src/test/java/com/znsio/api/userPostsAndAlbumsAPI.feature

```diff
@@ -0,0 +1,36 @@
+@userPostsandAlbums @prod
+  #  TARGET_ENVIRONMENT=prod TYPE=api TAG=@getUserPosts ./gradlew test
+
+  Feature: Fetch user details wrt jsonPlaceHolder api
+    Background:
+      * def jsonPostsData = read('classpath:com/znsio/templates/postsData.json')
```

Comment:
If these variable is not common throughout scenarios, initialise them in scenario level.